src/frontend/src/hooks/useBasePositionsContext.ts

```diff
@@ -0,0 +1,47 @@
+import { useState, useEffect } from 'react';
+import { useApiClients, BasePosition, combineUrls, useTelemetryLogger } from '@equinor/fusion';
+
+export type useBasePositionsContext = {
+  basePositions: BasePosition[];
+  isFetchingBasePositions: Boolean;
+  basePositionsError: Error | null
+}
+
+const useBasePositionsContext = (): useBasePositionsContext => {
```

Comment:
useBasePositions instead of useBasePositionsCONTEXT? Also: This could be a collection in the app reducer to cache them